sass/BUILD

```diff
@@ -11,5 +11,8 @@ exports_files([
 nodejs_binary(
     name = "sass",
     entry_point = "sass/sass.js",
-    node_modules = "@build_bazel_rules_sass_compiletime_deps//:node_modules",
+    install_source_map_support = False,
```

Comment:
This is a duplicate of #45, for what it's worth.

Reply:
Yeah, but this one has merge conflicts and it looks like it's not going to be merged soon. I can remove this line, in case you want the other PR to get in.

Reply:
Perhaps more pertinently, #45 is blocked on a build failure of unknown origin and it's likely that this will trigger the same issue.

Reply:
Looking at the CI status on this PR, it looks like it's the same failure I ran into in #55

Reply:
I've actually worked around this previously by just specifying `strip_prefix = "sass/`.